### PR TITLE
i18n(es): Update `reference/` `V3`

### PR DESCRIPTION
**Pull request by Waxer59**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
Update `reference/` `V3` [`PR`](https://github.com/withastro/docs/pull/4166/files#diff-db473265321c7d52cab488e5f6e46ed80df851e219da93e3b6aa30e0d704d5af)

### CODE REVIEW

File: src/content/docs/es/reference/cli-reference.mdx [hunk 1]
```diff
@@ -357,3 +352,129 @@ Abre automáticamente la aplicación en el navegador al iniciar el servidor.
...
+Similar a [`astro preview`](#astro-preview), inicia un servidor local para servir el directorio estático `dist/`.
+
+```js
+import { preview } from "astro";
+
+const previewServer = await preview({
+  root: "./my-project",
+});
+
+// Stop the server if needed
```
Comment (dreyfus92) [inline src/content/docs/es/reference/cli-reference.mdx:462]: Falta traducir

**Review by dreyfus92 — CHANGES_REQUESTED** [review]
Te dejo una pequeña sugerencia, buen trabajo @Waxer59 😁

File: src/content/docs/es/reference/cli-reference.mdx (hunk 1 above)
Comment (Waxer59) [inline src/content/docs/es/reference/cli-reference.mdx:462]:
```suggestion
// Detén el servidor si es necesario
```

**Comment by Waxer59** [comment]
Listo!, gracias por la revisión vaque

**Comment by Waxer59** [comment]
Commit applied: 
* [`c4d249`](https://github.com/withastro/docs/pull/4166/commits/c4d24997452599b85138582b2b3fa761753e76d8)

**Comment by Waxer59** [comment]
Commit (it has not been necessary to update the page):

* [`8190f7`](https://github.com/withastro/docs/pull/4166/commits/8190f7e1c737de336a0aad046b06164c733395e0)

File: src/content/docs/es/reference/adapter-reference.mdx [hunk 2]
```diff
@@ -189,3 +242,185 @@ Por lo general, puedes llamar a `app.render(request)` sin usar `.match` porque A
...
+}
+```
+
+:::caution
+`entryFile` es de tipo `URL` y representa la ruta física del archivo en el sistema de archivos. Esto significa que las rutas cambian según el sistema operativo en el que se ejecute el código.
+:::
+
+#### Entornos serverless
+
+Establecer `functionPerRoute:true` en un entorno serverless crea un archivo JavaScript (controlador) para cada ruta. El nombre de un controlador puede variar según la plataforma de alojamiento que estés utilizando: lambda, función, página, etc.
```
Comment (Waxer59) [inline src/content/docs/es/reference/adapter-reference.mdx:361]:
```suggestion
Establecer `functionPerRoute: true` en un entorno serverless crea un archivo JavaScript (controlador) para cada ruta. El nombre de un controlador puede variar según la plataforma de alojamiento que estés utilizando: lambda, función, página, etc.
```

File: src/content/docs/es/reference/adapter-reference.mdx [hunk 3]
```diff
@@ -189,3 +242,185 @@ Por lo general, puedes llamar a `app.render(request)` sin usar `.match` porque A
...
+
+Cada una de estas rutas está sujeta a un [arranque en frío](https://azure.microsoft.com/en-us/blog/understanding-serverless-cold-start/) cuando se ejecuta el controlador, lo que puede causar cierto retraso. Este retraso está influenciado por diferentes factores.
+
+Con `functionPerRoute: false`, solo hay un controlador único encargado de renderizar todas tus rutas. Cuando se activa este controlador por primera vez, estarás sujeto a un arranque en frío. Después de eso, todas las demás rutas deberían funcionar sin retraso. Sin embargo, perderás el beneficio de la división de código que proporciona `functionPerRoute:true`.
```
Comment (Waxer59) [inline src/content/docs/es/reference/adapter-reference.mdx:365]:
```suggestion
Con `functionPerRoute: false`, solo hay un controlador único encargado de renderizar todas tus rutas. Cuando se activa este controlador por primera vez, estarás sujeto a un arranque en frío. Después de eso, todas las demás rutas deberían funcionar sin retraso. Sin embargo, perderás el beneficio de la división de código que proporciona `functionPerRoute: true`.
```

**Comment by Waxer59** [comment]
Commit:
* [`9a8f0a`](https://github.com/withastro/docs/pull/4166/commits/9a8f0a3880bd75b89b4c1ff44e9c20c246e8b687)

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks @Waxer59 🙌

**Pull request merged**